docker-entrypoint.sh

```diff
@@ -2,6 +2,17 @@
 
 set -vex
 
+# Dashboard
+cd /dashboard
+REACT_APP_IASQL_ENV=local yarn start:build &
+
+# Run service
+cd /dashboard/run
+IASQL_ENV=local yarn start &
```

Comment:
Similarly, these should just use the env vars that were passed in to the Docker container. But won't block this PR on it.